runtime/litmus/src/lib.rs

```diff
@@ -783,29 +783,22 @@ impl pallet_sidechain::Config for Runtime {
 	type EarlyBlockProposalLenience = ConstU64<100>;
 }
 
-ord_parameter_types! {
-	pub const ALICE: AccountId = sp_runtime::AccountId32::new(hex!["d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"]);
-}
-
 impl pallet_identity_management::Config for Runtime {
 	type Event = Event;
 	type WeightInfo = ();
-	// TODO: use the real TEE account
-	type TEECallOrigin = EnsureSignedBy<ALICE, AccountId>;
+	type TEECallOrigin = EnsureEnclaveSigner<Runtime>;
 }
 
 impl pallet_identity_management_mock::Config for Runtime {
 	type Event = Event;
 	type ManageWhitelistOrigin = EnsureRoot<Self::AccountId>;
 	type MaxVerificationDelay = ConstU32<10>;
-	// TODO: use the real TEE account
-	type TEECallOrigin = EnsureSignedBy<ALICE, AccountId>;
+	type TEECallOrigin = EnsureEnclaveSigner<Runtime>;
```

Comment:
Looks good to me. But i  think it would be better for `pallet_identity_management_mock` to still use **Alice**, what do you think?